### PR TITLE
Modify Spatial Query Page 

### LINKED ISSUE #29: Temporal-Spatial Query Page Overhaul
Frontend: 

- [ ] Changes mentioned by GT
- [x] GIF for user interaction (asset - @jinamshah )
- [ ] datepicker: shared component? MUI component?
- [ ] draw button (`MapControls.js`) update - color + accordian
- [x] Grey out Next if all selections are not complete

Image

### PR DESCRIPTION
**Pull request by nisarg210**
- Split Page in 30:70 ratio
- Modify Map height to fit entire screen.
- Added Side Block.

### CODE REVIEW

**Comment by nisarg210** [comment]
New Page After Modifications.
image

**Review by jinamshah — CHANGES_REQUESTED** [review]
* I've uploaded the gif for spatial query demo `src/assets/gifs/spatial_query_eg.gif`
* The left side and right side backgrounds are misaligned
Screenshot 2024-11-25 at 1 28 49 PM
* move the date selection lower - leaves a lot of space at the bottom
* box drawn on the spatial query page doesn't change orientation if the map is rotated
* users should not be allowed to draw multiple boxes on the map
* left and right sides need additional padding - refer to the figma design for context

**Comment by jinamshah** [comment]
++ needs rebase with base branch - https://github.com/precision-sustainable-ag/drone-image-manipulation/network

File: src/pages/SpatialQuery/map.js
Comment (akshat22) [inline src/pages/SpatialQuery/map.js:1]: Current implementation has extra space in the left and right.

Refer to this file for page layout UI:
https://github.com/precision-sustainable-ag/drone-image-manipulation/blob/feature/mission-selection-changes/src/pages/orthomosaic_display/App.js

File: src/assets/gifs/spatial_query_eg.gif
Comment (akshat22) [inline src/assets/gifs/spatial_query_eg.gif:1]: delete this since you are using the video file

File: src/pages/SpatialQuery/map.js [hunk 1]
```diff
@@ -1,302 +1,345 @@
...
+        center: fromLonLat([-78.99, 35.43]),
+        zoom: 9,
+      })
+    );
+  }, []);
 
-    return (
-        <Box
+  const drawArea = (source, map) => {
+    console.log(map.getView().getProjection());
```
Comment (akshat22) [inline src/pages/SpatialQuery/map.js]: remove print

**Review by akshat22 — CHANGES_REQUESTED** [review]

File: src/pages/SpatialQuery/map.js
Comment (nisarg210) [reply src/pages/SpatialQuery/map.js:1]: According to figma design, we need extra padding on left and right both. As this is different from all other pages. @jinamshah also suggested to add more padding.

**Comment by jinamshah** [comment]
* map draw box problem is still not fixed (see review above)
* remove padding - 40-60 split looks fine with the video present
* map gets cut by the footer

**Review by jinamshah — APPROVED** [review]

**Pull request merged**